tests/compute/test_sampling.py

```diff
@@ -272,6 +272,7 @@ def _gen_neighbor_sampling_test_graph(hypersparse, reverse):
         }, {'user': card if card is not None else 4})
         g = g.to(F.ctx())
         g.edata['prob'] = F.tensor([.5, .5, 0., .5, .5, 0., 1.], dtype=F.float32)
+        g.edata['mask'] = F.tensor([True, True, False, True, True, False, True])
```

Comment:
`mask` should be in `int8_t` or `uint8_t. And if `F.tensor([0, 1, 0, 1, 0]) is passed in, program will crash in `ATEN_FLOAT_INT8_UINT8_TYPE_SWITCH`?
Though this is rare case(maybe mask data is loaded from numpy data file and forget to format it), is it possible to throw exception in python or always format to supported dtypes internally?

Reply:
It will report an error if the data type is wrong.

I don't think throwing exception in Python (or correcting the format internally) is a good idea since it introduces extra overhead for rare cases.  A clear error message should be enough.  What is your concern?

Reply:
if a clear error message is given, that's enough. what does the error message look like? can user figure out that dtype of `g.edata['mask']` is incorrrect?